Bottom: f65c2cb86cd9929c502023324264112e2888169f
Top:    c05d2765035a96372825abf8b6a3c94164f65a99
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 01:21:42 +0100

Refresh of draw-c-draw-all-vias-under-the

---

diff --git a/src/draw.c b/src/draw.c
index 37c1dfe..9017f92 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -491,7 +491,6 @@ DrawEverything (BoxTypePtr drawn_area)
 		    &plated);
 	}
     }
-#if 1
   /* Draw the solder mask if turned on */
   if (gui->set_layer ("componentmask", SL (MASK, TOP), 0))
     {
@@ -507,7 +506,6 @@ DrawEverything (BoxTypePtr drawn_area)
       DrawMask (drawn_area);
       SWAP_IDENT = save_swap;
     }
-#endif
   /* Draw top silkscreen */
   if (gui->set_layer ("topsilk", SL (SILK, TOP), 0))
     DrawSilk (0, COMPONENT_LAYER, drawn_area);
